analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AvoidUnsealedAttributes.vb

```diff
@@ -0,0 +1,27 @@
+﻿Public Class PublicAttribute ' Noncompliant {{Seal this attribute or make it abstract.}}
```

Comment:
There's a problem with the way how our test framework works. We can't assert locations `^^^^` on the first line.
Moving the problem to 2nd line will fix the issue.
```suggestion
' Test framework can't assert location on 1st line
﻿Public Class PublicAttribute ' Noncompliant {{Seal this attribute or make it abstract.}}
```